From 3c2b179300b6f3b5619400acb65d05ecd08c4c57 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Thu, 15 Oct 2020 18:52:26 -0400
Subject: [PATCH] audio: lib: windows: Add <cctype> header for std::isdigit.

Previously the header must have been pulled in by one of the Boost
includes, because with a newer Boost version I had build errors. This
makes the include explicit so std::isdigit is guaranteed to be declared.
---
 gr-audio/lib/windows/windows_sink.cc   | 1 +
 gr-audio/lib/windows/windows_source.cc | 1 +
 2 files changed, 2 insertions(+)

diff --git a/gr-audio/lib/windows/windows_sink.cc b/gr-audio/lib/windows/windows_sink.cc
index 9c6026530..8d66b245e 100644
--- a/gr-audio/lib/windows/windows_sink.cc
+++ b/gr-audio/lib/windows/windows_sink.cc
@@ -22,6 +22,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <cctype>
 #include <iostream>
 #include <sstream>
 #include <stdexcept>
diff --git a/gr-audio/lib/windows/windows_source.cc b/gr-audio/lib/windows/windows_source.cc
index 465c62f02..8004a4b2f 100644
--- a/gr-audio/lib/windows/windows_source.cc
+++ b/gr-audio/lib/windows/windows_source.cc
@@ -22,6 +22,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <cctype>
 #include <iostream>
 #include <sstream>
 #include <stdexcept>
-- 
2.20.1

